assets/js/WfParameter.js

```diff
@@ -409,5 +413,6 @@ WfParameter.propTypes = {
   changeParam:      PropTypes.func.isRequired,
 	getParamText:     PropTypes.func.isRequired,
   setParamText:     PropTypes.func.isRequired,
-	toggleDrag:       PropTypes.func.isRequired  
+  startDrag:        PropTypes.func.isRequired,
```

Comment:
This is fine; in the future you could always make this setDrag(true or false) to reduce this to one function.